network/endpoint.go

```diff
@@ -130,7 +130,7 @@ func (endpoint *EndpointInfo) GetHostComputeEndpoint() *hcn.HostComputeEndpoint
 		HostComputeNetwork:   endpoint.NetworkID,
 		HostComputeNamespace: endpoint.NamespaceID,
 		Dns: hcn.Dns{
-			Suffix:     endpoint.DNS.Suffix,
+			Domain:     endpoint.DNS.Suffix,
```

Comment:
This is fine for now if you only have one entry in the cni config search list, but this should actually be search instead of domain. That involves making a change in cni.go as well from a joined list to just a list. If you're interested you can look at my other commit in master for reference